bodhi/tests/server/services/test_updates.py

```diff
@@ -3831,10 +3879,74 @@ def test_batched_update(self, publish, *args):
         resp = self.app.post_json('/updates/', args)
         up = self.db.query(Update).filter_by(title=resp.json['title']).one()
         up.builds[0].ci_status = CiStatus.passed
+        up.comment(self.db, u"foo1", 1, u'foo1')
+        up.comment(self.db, u"foo2", 1, u'foo2')
         self.db.commit()
+
         resp = self.app.post_json(
             '/updates/%s/request' % args['builds'],
             {'request': 'batched', 'csrf_token': self.get_csrf_token()})
+
         self.assertEqual(resp.json['update']['request'], 'batched')
         publish.assert_called_with(
             topic='update.request.batched', msg=mock.ANY)
+
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_newpackage_update_bypass_batched(self, publish, *args):
+        """
+        Make sure a security update skips the 'batched' request and immediately enters stable
```

Comment:
We should correct this docblock to say "new package update" instead of security.